modules/administration-guide/partials/proc_configuring-che-with-custom-resources-to-use-a-rest-backup-server.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-{prod-id-short}-with-custom-resources-to-use-a-rest-backup-server_{context}"]
+= Configuring {prod-short} with custom resources to use a REST backup server
+
+To configure {prod-short} to use a REST backup server:
+
+.Prerequisites
+
+* Configured REST backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+. Create a secret containing the `repo-password` key with a Base64 encoded password:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-encryption-password-secret
+  namespace: eclipse-che
+type: Opaque
+stringData:
+  repo-password: __<base64_encoded_password>__
+----
++
+(The {prod-short} Operator setup up this password for the backup repository that the {prod-short} Operator will create from this custom object on the backup server.)
++
+include::snip_warning-about-losing-backup-repo-password.adoc[]
+
+. Create a secret (for example, `name: rest-server-auth-secret`) containing the `username` and `password` keys for the REST server user credentials, using Base64 encoded username and password values.
++
+NOTE: Skip this step if not using the REST server user credentials.
+
+. Create the `CheBackupServerConfiguration` custom object:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheBackupServerConfiguration
+metadata:
+  name: backup-server-configuration
+  namespace: {prod-namespace}
+spec: <1>
+  rest:
+    protocol: https <2>
+    hostname: _<hostname>_ <3>
+    port: 1234 <4>
+    repositoryPath: {prod-id}-backups <5>
+    repositoryPasswordSecretRef: backup-encryption-password-secret <6>
+    credentialsSecretRef: rest-server-auth-secret <7>
+----
+<1> `spec` property of this custom object must only have one section configured.
+<2> Optional property that specifies the protocol to be used: `https` is the default value; `http` is a permitted value.
+<3> Backup server hostname.
+<4> Optional property that specifies the port on which the backup server is running. The default value is `8000`.
+<5> Path on the backup server where the backup snapshots are stored.
+<6> Secret created in step 1.
+<7> Secret created in step 2.
```

Comment:
Please add a note, that the field is optional.

Reply:
Added. Thank you.